src/oscar/templates/oscar/dashboard/offers/progress.html

```diff
@@ -11,9 +11,9 @@ <h3>{% trans "Steps" %}</h3>
                 {% trans "1. Name and description" %}
             </a>
             {% else %}
-            <span class="{% if step == 1 %}active{% else %}disabled{% endif %}">
+            <a class="nav-link {% if step == 1 %}active{% else %}disabled{% endif %}">
```

Comment:
Maybe you can use `.text-reset` with `.disabled`, so that it does not look like a link?

Reply:
Yep, done.